plugins/woocommerce/tests/e2e-pw/global-setup.js

```diff
@@ -75,6 +76,14 @@ module.exports = async ( config ) => {
 				.storageState( { path: process.env.ADMINSTATE } );
 			console.log( 'Logged-in as admin successfully.' );
 			adminLoggedIn = true;
+
+			if (ENABLE_HPOS === undefined || ENABLE_HPOS === '0') {
+				console.log( 'Trying to switch off HPOS...' );
+				await adminPage.goto('/wp-admin/admin.php?page=wc-settings&tab=advanced&section=features');
+				await adminPage.getByLabel('WordPress posts storage (legacy)').check();
+				await adminPage.getByRole('button', { name: 'Save changes' }).click();
+				console.log('HPOS Switched off successfully');
```

Comment:
It would be good to have an explicit check that verifies if HPOS was indeed disabled or not, before displaying the message "HPOS Switched off successfully" to the tester. 

This is because we've had e2e flakiness in the past where a test clicks on the "save" button and then immediately proceeded to the next action, which interrupted the save action, and therefore causing flakiness later on in the test. We have to make sure the save action was completed first before proceeding.

The check could be a simple `GET /wp-json/wc/v3/settings/advanced/woocommerce_custom_orders_table_enabled` REST API call, and verifying if `value` was set to `no` in the response.

Any UI check will do too, like verifying  if the  "WordPress posts storage (legacy)" radio button was indeed in a selected state after the page reload resulting from the "Save changes" click.

Or I guess merely waiting for this POST call, which triggers right after clicking "Save changes", to return `200` is enough to signify that the save action was completed:

<img width="473" alt="Screenshot 2023-10-02 at 12 16 52 PM" src="https://github.com/woocommerce/woocommerce/assets/4509348/73277105-472d-49df-91f7-6ab978a65e43">

If you'll go with the approach above, you'll have to use Playwright's [network monitoring](https://playwright.dev/docs/network#network-events) to verify the response like what we did [here](https://github.com/woocommerce/woocommerce/blob/f73b8a80f6d47a871bed01d8d288cf29e0ab5bfe/plugins/woocommerce/tests/e2e-pw/tests/admin-analytics/analytics-overview.spec.js#L256-L259).

Reply:
I see that you're now using the REST API to enable or disable HPOS which is really cool. Code now looks cleaner and more concise. Thanks a lot!